ios/RNFirebase/messaging/RNFirebaseMessaging.m

```diff
@@ -126,6 +126,20 @@ - (void)messaging:(nonnull FIRMessaging *)messaging
     }
 }
 
+RCT_EXPORT_METHOD(getAPNSToken:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    NSData *apnsToken = [FIRMessaging messaging].APNSToken;
+    if (apnsToken) {
+        const char *data = [apnsToken bytes];
+        NSMutableString *token = [NSMutableString string];
+        for (NSInteger i = 0; i < apnsToken.length; i++) {
+            [token appendFormat:@"%02.2hhX", data[i]];
+        }
+        resolve([token copy]);
+    } else {
+        reject(@"messaging/apns-token-error", @"Failed to retrieve APNS token.", nil);
```

Comment:
Not sure this should be an error, could we have the function always resolve, with a token or with null, e.g. changing this to `resolve([NSNull null]);`